scenarios/NGSIM/README.md

```diff
@@ -16,6 +16,10 @@ Once that is done, you should be able to build your scenarios in the normal way,
 ```bash
 scl scenario build-all --clean scenarios/NGSIM
 ```
+or the following command to disable generating `shifted_map-AUTOGEN.net.xml`
```

Comment:
I would reword this to:  "or use the `--allow-offset-map` option to prevent auto-shifting the map to the origin, like:  "

(The main reason for the rewording is because the filename `shifted_map-AUTOGEN.net.xml` is encapsulated, i.e., not to be known or cared about by users.)